src/services/Sso.php

```diff
@@ -148,79 +167,99 @@ function afterDeleteUser($userName){
+        }                
         $js = <<<JS
-      (  
-       function renderEmbeddedHtmlWithAuthtoken()
-      {  var embedUrl  = "{$forumUrl}";   
-         var userStatus = "{$userStatus}";
-
-         var cookieForumLogoutToken = "{$cookieForumLogoutToken}";
-        var wtbWrap = document.createElement('div');
-        wtbWrap.id = "wtEmbedCode";
-        var embedScript = document.createElement('script');
-        embedScript.id = "embedded_forum";
-        embedScript.type = 'text/javascript'; 
-        if(typeof cookieForumLogoutToken != 'undefined' && cookieForumLogoutToken != 0){ 
-            if(userStatus == 'loggedIn'){
-                embedUrl += "/js/mb/embed.js?authtoken="+cookieForumLogoutToken;
-            }else{
-                embedUrl += "/js/mb/embed.js?authtoken=0";
+          (  
+           function renderEmbeddedHtmlWithAuthtoken()
+          { var embedUrl  = "{$forumUrl}";
+            var userStatus = "{$userStatus}";            
+            var wtbWrap = document.createElement('div');
+            wtbWrap.id = "wtEmbedCode";            
+            var embedScript = document.createElement('script');
+            embedScript.id = "embedded_forum";
+            embedScript.type = 'text/javascript'; 
+            embedUrl += "/js/mb/embed.js";                        
+            embedScript.src = embedUrl;
+            embedScript.setAttribute('data-version','1.1');
+            wtbWrap.appendChild(embedScript);
+            if(document.getElementById('wtEmbedCode') != null){
+                document.getElementById('wtEmbedCode').innerHTML = '';
+                document.getElementById('wtEmbedCode').appendChild(embedScript);                  
             }
-        } else{ 
-            embedUrl += "/js/mb/embed.js?authtoken=0";
-        }
-        embedScript.src = embedUrl; 
-        wtbWrap.appendChild(embedScript); 
-        document.getElementById('embedForum').appendChild(embedScript);
-
-      })();
-JS;
-      return $js ;
-    }   
-  function renderJsScriptUnEmbedded(){  
+          })();
+        JS;
+        return $js;
+   }   
+  function renderJsScriptUnEmbedded(){    
+        $baseUrl = UrlHelper::siteUrl();        
+        $token = Craft::$app->getSession()->get(Craft::$app->getUser()->tokenParam);
+        $forumUrl = Craft::$app->getProjectConfig()->get('plugins.websitetoolboxforum.settings.forumUrl',false);        
         if(isset($_COOKIE['forumLogoutToken'])){
             $cookieForumLogoutToken = $_COOKIE['forumLogoutToken'];
+            if(!isset($_COOKIE['logInForum'])){
+                setcookie('logInForum', 1, time() + 3600,"/");                                
+                $_COOKIE['logInForum'] = 1;
+                echo '<img src='.$forumUrl.'/register/dologin?authtoken='.$cookieForumLogoutToken.' style="width:0px !important;height:0px !important;" border="0" alt="" id="imageTag">'; 
```

Comment:
style="width:0px !important;height:0px !important;" not required.

Reply:
Removed